sources/HUBComponentModelBuilderImplementation.m

```diff
@@ -437,11 +467,23 @@ - (id)copyWithZone:(nullable NSZone *)zone
     for (NSString * const customImageIdentifier in self.customImageDataBuilders) {
         copy.customImageDataBuilders[customImageIdentifier] = [self.customImageDataBuilders[customImageIdentifier] copy];
     }
-    
+
     for (NSString * const childIdentifier in self.childBuilders) {
-        copy.childBuilders[childIdentifier] = [self.childBuilders[childIdentifier] copy];
+        HUBComponentModelBuilderImplementation *childBuilder = self.childBuilders[childIdentifier];
```

Comment:
Suggestion: Merge this statement with the one on the next line, making it:
```objective-c
HUBComponentModelBuilderImplementation *childBuilder = [self.childBuilders[childIdentifier] copy];
```

Not just for style reasons, but makes the code a bit more "safe", as you might not accidentially use `childBuilder` when it really shouldn't be used at all in this scope.